carma_cooperative_perception/docs/stack_design.md

```diff
@@ -0,0 +1,71 @@
+# CARMA cooperative perception - Stack design
+
```

Comment:
Just a thought. @adamlm Maybe we want to limit the documentation for each PR to what is actually implemented in the PR. My intention is just to make sure are able to effectively review both documentation and implementation with each PR. What do you think?

Reply:
I agree for the Package Design documentation, but I think the Stack Design documentation should remain because it shows a more complete picture. I copied the stack documentation from the Confluence page. We can add a disclaimer under the necessary stack variants that they are unsupported due to the currently-implemented ROS Nodes. I will remove the links to unimplemented ROS Nodes.

How does that sound?